V1/auth/validate.py

```diff
@@ -0,0 +1,58 @@
+from flask import make_response, jsonify
+import re
+
+
+def response(status, message, status_code):
```

Comment:
Have a single-line docstring comment that tells the person reading your code what params the function takes and what it returns.